lint/queue.py

```diff
@@ -1,12 +1,15 @@
-import sublime
-
 from . import persist
 
+from collections import defaultdict
 import time
+import threading
 
+# TODO: Both `timers` and `running` herein grow unbounded. T.i. a proper
```

Comment:
Shouldn't we do something about this now?

Reply:
No, it's a good solution and a hotfix. Holding 100 or so locks should be no problem. Don't know. Is it?

It's one of the moving parts. Can we make the debouncer self-contained, so the debouncer decides when to clear up old timers, or do we need to trigger the clearance from outside? 

This async/parallel stuff just takes a week to sink. This is really hard to think about. At least for me.

Reply:
Okay, should I just do it? I add it to `def on_pre_close(self, view)`?

Reply:
> This async/parallel stuff just takes a week to sink. This is really hard to think about. At least for me.

Definitely.

Reply:
> Okay, should I just do it?

Add the cleanup to `on_pre_close` you mean? Doesn't sound like a bad idea.

Reply:
Ah, let's do it